gradle/shared.gradle

```diff
@@ -90,6 +90,8 @@ tasks.withType(Test) {
         reports.junitXml.destination = new File(System.env.CIRCLE_TEST_REPORTS, it.getName())
     }
 
+    enableAssertions = true
```

Comment:
Yup, assertions dont really add any additional cost except the evaluation of boolean expressions. Build times should be a major concern here.